src/Json/JsonContext.php

```
@@ -131,33 +137,23 @@ public function theJsonNodeShouldExist($jsonNode)
      *
      * @Given /^the JSON node "(?P<jsonNode>[^"]*)" should not exist$/
      */
-    public function theJsonNodeShouldNotExist($jsonNode)
+    public function theJsonNodeShouldNotExist($jsonNode): void
     {
-        $e = null;
-
         try {
             $realValue = $this->evaluateJsonNodeValue($jsonNode);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$realValue'. (undefined)

(UnusedLocalVariable)